File: SDKOCDemo/ViewController.m
//
//  ViewController.m
//  SDKOCDemo
//
//  Created by moubuns on 2018/12/12.
//  Copyright © 2018 FoxOne. All rights reserved.
//

#import "ViewController.h"
#import "FoxAsset.h"
#import "SDKOCDemo-Swift.h"



@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    [FoxPaySDKHelper getAssetWithCompleted:^(NSArray<FoxAsset *> * _Nonnull assets) {
        [assets enumerateObjectsUsingBlock:^(FoxAsset * _Nonnull obj, NSUInteger idx, BOOL * _Nonnull stop) {
            NSLog(@"%@", obj.name);
        }];
    }];
}


@end

File: SDKDemo/ViewController.swift
//
//  ViewController.swift
//  SDKDemo
//
//  Created by moubuns on 2018/9/3.
//  Copyright © 2018 FoxOne. All rights reserved.
//

import UIKit
import FoxOnePaySDK

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        
        PaySDKService.getAssets { completion in
            switch completion {
            case .success(let assets):
                print(assets)
            case .failure(let error):
                print(error)
                break
            }
        }
        
        PaySDKService.getUser { (completion) in
            switch completion {
            case .success(let user):
                print(user)
            case .failure(let error):
                print(error)
                break
            }
        }
    }
}

